components/bluetooth_proxy.rst

```diff
@@ -14,13 +14,21 @@ and Home Assistant.
 If you're looking to create an ESPHome node that is just a Bluetooth Proxy, see
 our `Bluetooth Proxy installer <https://esphome.github.io/bluetooth-proxies/>`__ website.
 
-.. note::
+.. warning:: Active connections
 
     The Bluetooth proxy of ESPHome provides Home Assistant with a maximum number of 3 simultaneous active connections.
     Devices which maintain a *continuous active* connection will consume one of these constantly, whilst devices which
     do *periodic disconnections and reconnections* will permit using more than 3 of them (on a statistical basis).
     Passively broadcasted sensor data (that is advertised by certain devices without active connections) is received
     separately from these, and is not limited to a specific number.
+
+    The :doc:`esp32` component should be configured to use the ``esp-idf`` framework, as the ``arduino`` framework
+    uses significantly more memory and performs poorly with the Bluetooth proxy enabled. When switching from
+    ``arduino`` to ``esp-idf``, make sure to update the device with a serial cable as the partition table is
```

Comment:
I'd be a bit more permissive an exact:

When switching from  ``arduino`` to ``esp-idf``, and using active connections, it is recommended to to update the device with a serial cable

because this is not something mandatory when somebody is using the device passively (which is default).

Reply:
I think this ok as-is given its in the `Active connections` block